thunder/executors/nvfuserex_impl.py

```diff
@@ -2093,7 +2093,11 @@ def var_mean(
 def _copy__check(
     copy_from: TensorProxy,
     copy_to: TensorProxy,
+    *,
+    grad_enabled: bool,
 ) -> bool:
+    if grad_enabled:
```

Comment:
Effectively it would mean raising a warning for every invocation of batch norm training. We should raise warnings if they are useful to users, here it would be just noisy. Let's not do that for now, but there's a path of raising a real error if we can query `is_leaf` on input tensor proxies.